encrypted-media/scripts/check-hdcp-policy.js

```diff
@@ -0,0 +1,26 @@
+function runTest(config, qualifier)
+{
+  function checkHdcpPolicy(hdcpPolicy)
+  {
+    return navigator.requestMediaKeySystemAccess(config.keysystem, getSimpleConfiguration())
+        .then(function(access) {
+          return access.createMediaKeys();
+        })
+        .then(function(mediaKeys) {
+          // As HDCP policy depends on the hardware running this test,
+          // don't bother checking the result returned as it may or
+          // may not be supported. This simply verifies that
+          // getStatusForPolicy() exists and doesn't blow up.
+          return mediaKeys.getStatusForPolicy({minHdcpVersion: hdcpPolicy});
```

Comment:
This might return a rejected promise, e.g. for clear key. Will this cause the test to fail? For the context, see https://github.com/WICG/hdcp-detection/issues/2

Reply:
The test will fail, but if you look at the results you will see the failure due to error NotSupportedException. I think this is better for developers as they will see that it is not supported on the platform (by clear key, for example) and recognize they need to handle the failure. This is similar to the persistent-session tests (that are optional, but it is useful to know that persistent sessions aren't supported).